catalogs/catalog-messaging-kafka/src/main/java/com/datastrato/gravitino/catalog/kafka/KafkaCatalogPropertiesMetadata.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.kafka;
+
+import com.datastrato.gravitino.catalog.BaseCatalogPropertiesMetadata;
+import com.datastrato.gravitino.catalog.PropertyEntry;
+import java.util.Collections;
+import java.util.Map;
+
+public class KafkaCatalogPropertiesMetadata extends BaseCatalogPropertiesMetadata {
+
+  // The "bootstrap.servers" property specifies the Kafka broker(s) to connect to, allowing for
+  // multiple brokers by comma-separating them.
+  public static final String BOOTSTRAP_SERVERS = "bootstrap.servers";
+
+  private static final Map<String, PropertyEntry<?>> KAFKA_CATALOG_PROPERTY_ENTRIES =
+      Collections.singletonMap(
+          BOOTSTRAP_SERVERS,
+          PropertyEntry.stringRequiredPropertyEntry(
+              BOOTSTRAP_SERVERS,
+              "The Kafka broker(s) to connect to, allowing for multiple brokers by comma-separating them",
+              true /* immutable */,
+              false /* hidden */));
+
```

Comment:
Does this only include one attribute, with the other attributes to be added in a later PR?

Reply:
yes, other properties' metadata will be added in a later PR